JS/setinterval.js

```diff
@@ -0,0 +1,5 @@
+function myFunction() {
+  setInterval(function() {
+    alert("Hello");
```

Comment:
Change this to the clock example we discussed.

Reply:
I added a new file for it .Its done